api_app/views.py

```diff
@@ -148,54 +206,63 @@ def ask_analysis_availability(request):
     )
 
     serializer = JobAvailabilitySerializer(
-        data=data_received, context={"request": request}
+        data=data_received, context={"request": request}, many=True
```

Comment:
yeah, you are right, you got another point of code that require changes.

You mentioned that this would not work for requests with a single observable, am I right? So this would be a breaking change or I am getting this wrong?

In any case, an additional test would clarify this. (right now there is a test `test_ask_analysis_availability` for this. We could add the one for the multiple observables too)

Reply:
In the new model, there's no concept of a "single observable". It's still a list of multiple observables. Just the length of the list is 1